Cue/app/tabs/library/LibraryHome.js

```diff
@@ -76,10 +80,27 @@ class LibraryHome extends React.Component {
   componentWillReceiveProps(newProps: Props) {
     this.setState({
       refreshing: false
-    })
+    })
+
+    if (newProps.inaccessibleDecks && newProps.inaccessibleDecks.length > 0) {
+      newProps.inaccessibleDecks.forEach(deck => {
+        Alert.alert(
+          (Platform.OS === 'android' ?
+            'The deck “' + deck.name + '” is no longer available from the original owner' :
+            'The Deck “' + deck.name + '” Is No Longer Available From The Original Owner'),

```

Comment:
Well... our deck name limit is 255 characters. So I don't any amount of shortening absolutely fix truncation hergie. Maybe put the deck title in the alert description? Not sure if that truncates as well